### PR TITLE
No bytecodecompare if tests failed (issue #2300)

### PR DESCRIPTION
**Pull request by limexp**
AppVeyor stops tests at first failure. This is opposite to Travis behavior.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -183,9 +183,9 @@ before_script:
       && scripts/create_source_tarball.sh)
 
 script:
-    - test $SOLC_EMSCRIPTEN != On || (scripts/test_emscripten.sh)
+    - test $SOLC_EMSCRIPTEN != On || (scripts/test_emscripten.sh) || SOLC_STOREBYTECODE=Off
```
Comment (chriseth) [inline .travis.yml]: That's a very good idea, but won't it make the travis build succeed if the tests fail?

Comment (limexp) [reply .travis.yml]: Thank you! Great point. fixed

**Review by chriseth — APPROVED** [review]

**Comment by chriseth** [comment]
Thanks!

**Comment by axic** [comment]
@chriseth this seems to have broken Appveyor:
```
*** No errors detected
if ($env:priv_key) { scripts\bytecodecompare\storebytecode.bat $Env:CONFIGURATION $bytecodedir }
scripts\bytecodecompare\storebytecode.bat : The module 'scripts' could not be loaded. For more information, run 'Import-Module scripts'.
At line:1 char:22
+ ... f ($env:priv_key) { scripts\bytecodecompare\storebytecode.bat $Env:CO ...
+                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (scripts\bytecod...orebytecode.bat:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CouldNotAutoLoadModule
 
Command executed with exception: The module 'scripts' could not be loaded. For more information, run 'Import-Module scripts'.
```

**Comment by limexp** [comment]
@axic 
this is my fault((( 
missed additional ```cd %APPVEYOR_BUILD_FOLDER%```

**Pull request merged**